packages/dappmanager/src/modules/https-portal/migration.ts

```diff
@@ -34,6 +34,19 @@ export async function addAliasToRunningContainersMigration(): Promise<void> {
       migrateCoreNetworkInCompose(container);
       addNetworkAliasCompose(container, dncoreNetworkName, [alias]);
 
+      // Wifi and VPN containers needs a refresh connect due to its own network configuration
+      if (
+        container.containerName === params.vpnContainerName ||
+        container.containerName === params.wifiContainerName
+      ) {
+        await shell(`docker rm ${containerName} --force`);
+        await shell(
+          `docker-compose -f ${params.DNCORE_DIR}/${
+            container.dnpName.split(".")[0]
+          } up -d`
```

Comment:
You should re-use the docker-compose wrappers from modules/docker https://github.com/dappnode/DNP_DAPPMANAGER/blob/cb58825c4a/packages/dappmanager/src/modules/docker/compose/cli.ts